virtool/analyses/data.py

```diff
@@ -107,14 +114,11 @@ async def get(
         if document is None:
             raise ResourceNotFoundError()
 
-        if if_modified_since is not None:
-            try:
-                updated_at = document["updated_at"]
-            except KeyError:
-                updated_at = document["created_at"]
+        aws = []
+
+        aws.append(check_if_analysis_modified(if_modified_since, document))
 
-            if if_modified_since == updated_at:
-                raise ResourceNotModifiedError()
+        await wait_for_checks(*aws)
```

Comment:
This can be collapsed into one readable line without the declaration of a new list. Make sure you understand how `*` works.